homeassistant/components/camera/onvif.py

```diff
@@ -33,6 +35,22 @@
 DEFAULT_PASSWORD = '888888'
 DEFAULT_ARGUMENTS = '-q:v 2'
 
+ATTR_PAN = "pan"
+ATTR_TILT = "tilt"
+ATTR_ZOOM = "zoom"
+
+DIR_UP = "UP"
+DIR_DOWN = "DOWN"
+DIR_LEFT = "LEFT"
+DIR_RIGHT = "RIGHT"
+ZOOM_OUT = "ZOOM_OUT"
+ZOOM_IN = "ZOOM_IN"
+
+SERVICE_PTZ = "ptz"
```

Comment:
Service names for platforms need to be prefixed with the platform name. So name it `onvif_ptz`